tmt/steps/__init__.py

```diff
@@ -2096,6 +2101,123 @@ class Plugin(BasePlugin[StepDataT, PluginReturnValueT]):
     Common parent of all step plugins that do work against a particular guest
     """
 
+    @overload
+    def write_command_report(
+        self,
+        *,
+        path: Path,
+        label: str,
+        timer: Stopwatch,
+        command: Optional[Union[Command, ShellScript]] = None,
+        output: CommandOutput,
+        exc: None = None,
+    ) -> None:
+        pass
+
+    @overload
+    def write_command_report(
+        self,
+        *,
+        path: Path,
+        label: str,
+        timer: Stopwatch,
+        command: Optional[Union[Command, ShellScript]] = None,
+        output: None = None,
+        exc: RunError,
+    ) -> None:
+        pass
+
+    def write_command_report(
+        self,
+        *,
+        path: Path,
+        label: str,
+        timer: Stopwatch,
+        command: Optional[Union[Command, ShellScript]] = None,
+        output: Optional[CommandOutput] = None,
+        exc: Optional[RunError] = None,
+    ) -> None:
+        """
+        Format and write a command output into a report file.
+
+        To provide unified look of various files reporting command
+        outputs, this helper would combine its arguments and write the
+        report into a given file. The file is created if it does not
+        exist yet, and the report will be added to the content of the
+        file, not overwriting it.
+
+        The report is generated by :py:func:`render_command_report`.
```

Comment:
```suggestion
        The report is generated by :py:func:`tmt.utils.render_command_report`.
```
Needs to be fully-qualified unless it is in the same context (sibling method/attribute etc., same module, etc.)
https://tmt--4263.org.readthedocs.build/en/4263/code/autodocs/tmt.steps.html?readthedocs-diff=true&readthedocs-diff-chunk=1#tmt.steps.Plugin.write_command_report

Reply:
Ok, fixed in f7f74136, although I have a feeling it may work in some cases.